### ISSUE TITLE
Country Code Netherlands Antilles 'AN' Not Working

### ISSUE THREAD

**Issue by brpsmith** [catamphetamine/react-phone-number-input#179]
First - thank you for this component.  Very impressive and very helpful.  I believe a country code is missing but I am unable to find where in the code the culprit might be.  Setting the props - country, countries, and countryOptions to ' AN' the component fails to render a flag and in general causes other issues.   Here is the component declaration for reference:

                            ```
<PhoneInput
  value={this.getCellPhone()}
  id='input-cell'  
  name='input-cell'
  ref='cellPhone' 
  international={false}
  countries={['AN']}
  countryOptions={['AN']}
  country={'AN'}
  onChange={this.phoneInputChange.bind(this)}
/>    
```

**Comment by catamphetamine** [comment]
@brpsmith 
What is the exact issue you're experiencing and what is the desired outcome.
And format the code correcly.

**Comment by brpsmith** [comment]
Thanks.  Not sure what is up with the editor for the code not formatting correctly but the outcome I want is for the country code AN to work.  It doesn't in my case.

**Comment by catamphetamine** [comment]
> Not sure what is up with the editor for the code not formatting correctly but the outcome I want is for the country code AN to work. It doesn't in my case.

Describe in more specifics with steps to reproduce, the desired outcome and the observed outcome.
Otherwise this issue will be closed as being non-informative.

**Comment by brpsmith** [comment]
When I attempt to use the 'AN' country code using the code below, the flag renders as a broken image with the text AN below it.   If I use a country code like 'US' with the code below, everything works as expected.  Note that react-phone-number-input is imported as expected as well as the style sheets and the component code is part of a render method within another component.

`<PhoneInput
    value={''}
    international={false}
    countries={['AN']}
    countryOptions={['AN']}
    country={'AN'}
    onChange={phone => this.setState({ phone })}/>`

If the code does not make sense, please let me know what is missing/or at issue.

The expected outcome is that the AN country code works like the US and other ISO country codes.

Attaching screen shots - hopefully they come through.

![screen shot 2018-09-06 at 2 55 37 pm](https://user-images.githubusercontent.com/19190882/45178944-1ef80d00-b1e5-11e8-87ac-8c37439a36b4.png)

![screen shot 2018-09-06 at 2 55 26 pm](https://user-images.githubusercontent.com/19190882/45178952-23bcc100-b1e5-11e8-82ff-d21bfad05d1b.png)

**Comment by catamphetamine** [comment]
There's no "AN" country
https://www.nationsonline.org/oneworld/country_code_list.htm

**Comment by brpsmith** [comment]
Not sure where the disconnect is.  It is listed on the page you shared.  Screen shot below.  In addition, Amazon supports the country code as a valid target for SMS messages (https://docs.aws.amazon.com/sns/latest/dg/sms_supported-countries.html) as part of their Cognito offering.
.  
![screen shot 2018-09-06 at 3 44 46 pm](https://user-images.githubusercontent.com/19190882/45181277-fd4e5400-b1eb-11e8-8f38-c1dfd7c5ff72.png)

**Comment by catamphetamine** [comment]
Ah, I see, they're not sorted alphabetically.
Ok then, there is "AN" country.

**Comment by catamphetamine** [comment]
Well, then you can report the issue to Google somehow if you want.
https://github.com/googlei18n/libphonenumber/blob/master/resources/PhoneNumberMetadata.xml
"AN" is missing from their metadata.
I guess it's just that they don't differentiate between Netherlands and AN Netherlands so the metadata for Netherlands most likely is the combined one.

**Comment by brpsmith** [comment]
Will do - thank you for your help.

**Comment by catamphetamine** [comment]
@brpsmith Actually, I'll reopen this, because we can do better — I should validate the `country` passed (and also `countries` and `countryOptions`). In case of `AL` it should have thrown `Country not found: "AL"`.

**Comment by catamphetamine** [comment]
Released `react-phone-number-input@2.2.3` which throws if country is not found in metadata.
